docs/book/installation.rst

```diff
@@ -74,8 +95,41 @@ browser.
 
 .. tip::
 
-    You can also use the Symfony built-in webserver by running the ``php app/console server:run --env dev``
-    command and then accessing ``http://localhost:8000``.
+    We strongly recommend using the Symfony abuilt-in webserver by running the
+    ``php app/console server:run --env dev``
```

Comment:
`server:start` is more handy (runs process in the background). Also we don't need `--env dev` since it's default value.